unit-tests-java8/src/test/java/org/eclipse/collections/test/bimap/UnsortedBiMapTestCase.java

```diff
@@ -49,23 +57,4 @@ default void Iterable_remove()
                         Sets.immutable.with(3, 1),
                         Sets.immutable.with(2, 1)));
     }
-
-    @Override
-    @Test
-    default void RichIterable_toString()
-    {
-        String string = this.newWith(3, 2, 1).toString();
-        Pattern pattern = Pattern.compile("^\\{\\d\\.\\d+(E-\\d)?=(\\d),"
-                + " \\d\\.\\d+(E-\\d)?=(\\d),"
-                + " \\d\\.\\d+(E-\\d)?=(\\d)\\}$");
-        Matcher matcher = pattern.matcher(string);
-        Assert.assertTrue(string, matcher.matches());
-
-        assertEquals(
-                Bags.immutable.with("1", "2", "3"),
-                Bags.immutable.with(
-                        matcher.group(2),
-                        matcher.group(4),
-                        matcher.group(6)));
-    }
```

Comment:
This was a lot of effort to make assertions about the values when the map contains random keys. The new test is just as good for toString() and covers the views as well.